build/style.css

```diff
@@ -100,3 +104,32 @@ strong {
 	opacity: 0.6;
 	border-bottom-color: #bb5536;
 }
+
+div.main.wrapper h2 {
+	color: #b30000;
+}
+
+div.wrapper h2.trouble {
+	float: right;
+	padding: 20px 0px;
+	color: #a9a9a9;
+	text-transform: uppercase;
+}
+div.article.full.section.content{
```

Comment:
Your selectors are too precise. No need to be that specific. I would change `div.article.full.section.content` to just `.section`.

That way, when you remove the `full` class from the html, your CSS doesn't need to change too. Because once you remove `"full"` from your HTML, this style rule will no longer work because of how specific it is.